src/client/app/components/HeaderButtonsComponent.tsx

```diff
@@ -174,8 +174,15 @@ export default function HeaderButtonsComponent() {
 								style={state.csvViewableLinkStyle}
 								disabled={state.shouldCSVButtonDisabled}
```

Comment:
This is not working as I would expect. It is supposed to gray out the current page. Given this, I think there needs to be state for each CSV page and not just one. NOte the route seems wrong above and that should be part of this change.